view/rooms/app.js

```diff
@@ -829,6 +838,32 @@ function closeConfirmationModal() {
   );
 }
 
+/**
+ * Room not found so we display error message
+ * @param {DOM} document - document to display not found modal on
+ */
+function displayNotFoundModal(doc) {
+  if(doc) {
+    doc.getElementById('create_room_button').onclick = () => window.location = '/rooms/new';
```

Comment:
since this is the same inside if and else, might be better extracting to a const. same for below line

Reply:
maybe extract the whole fn